roles/wisdom_deploy_model/files/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o errtrace
+set -x
+
+
+env
+PROMPT="- name: Install python-3.12 on RHEL"
+CONTEXT=""
+cd /etc/protos
```

Comment:
something went wrong around [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/705/pull-ci-openshift-psap-ci-artifacts-main-wisdom-load-test-light/1663953774378487808/artifacts/load-test-light/gather-extra/artifacts/pods/wisdom_wisdom-tester_wisdom-tester.log)
```
+ cd /etc/protos
+ grpcurl -plaintext -proto wisdomextservice.proto -d '{ "prompt": "- name: Install python-3.12 on RHEL", "context": "" }' -H 'mm-vmodel-id: ansible-wisdom' modelmesh-serving.wisdom.svc:8033 caikit.runtime.WisdomExt.WisdomExtService/AnsiblePredict
Failed to process proto source files.: could not parse given files: open wisdomextservice.proto: no such file or directory
```
maybe add a `ls` to know the content of the secrets?